### DISCUSSION TITLE
Versioning path hidden/inaccessible

### DISCUSSION THREAD

**Discussion by NatCoz** [MobiusSync/MobiusSync#209] [Q&A]
Hello there,

I'm using Mobius Ultimate v1.18.4 on iOS 15.5

Re: File Versioning path

When no file versioning path is designated, apparently the default path is .stversions, but on iOS, any file/folder name beginning with "." is hidden and inaccessible.

I can see Mobius created the .stversions folder and is storing files there. I'll be changing the path to "stversions", without the ".", so I can access these files via the Files app.

I can use Mobius to clean out the old .stversions folder before I make that move. But does anyone know of a way to delete the abandoned .stversions folder?

Thank you

**Comment by johannrichard** [comment]
Your best bet is using a third party app (e.g. like [File Browser Go](https://apps.apple.com/ch/app/filebrowsergo/id335493278?l=en)) which can display hidden folders and files, and can access third party app folders (like Möbius Sync's app folder) too.

**Comment by jackwasey** [comment]
Is anyone aware whether it would be possible for MobiusSync to use some iOS file browsing API, but deliberately enable hidden file viewing in some way? This is surely a widespread problem, with the wreckage of out-of-sync half-deleted .DS_Store and much else.

Or could MobiusSync have an option to never write hidden files to iOS devices, and either ignore them automatically bi-directionally? Getting in a state where a remote is trying to delete files that never existed is a mess already, and always ignored dot files are the cause.
